airbyte-integrations/connectors/source-github-singer/source_github_singer/source.py

```diff
@@ -37,18 +37,10 @@ class SourceGithubSinger(SingerSource):
     def _check_with_catalog(logger: AirbyteLogger, streams: List, config: json):
         repositories = config["repository"].split(" ")
         for repository in repositories:
-            org = repository.split("/")[0]
             # requests for checking streams permissions
             # first is for checking if user has access to Collaborators API
             # if user is not one of the collaborators, request will return 403 error
-            # third is for checking access and permission to Teams API
-            # Teams API is only available to authenticated members of the team's organization,
-            # in another case it will return 404 error
-            # if user doesn't have permission, it will return 401 error
-            check_streams = {
-                "collaborators": f"https://api.github.com/repos/{repository}/collaborators",
-                "teams": f"https://api.github.com/orgs/{org}/teams?sort=created_at&direction=desc",
-            }
+            check_streams = {"collaborators": f"https://api.github.com/repos/{repository}/collaborators"}
```

Comment:
can we do the same thing for the collaborators stream? Basically, if we're unable to access a stream because it doesn't apply to the repo at hand, instead of erroring we should: 
1. log a warning
2. inform the user in the docs that this is the expected behavior

The goal here is: If a user tries to sync data that doesn't apply to their data source, we should sync the data that does apply and just warn the user about the data that doesn't